library-best-practices/JLBP-10.md

```diff
@@ -30,7 +30,7 @@ accepting breaking changes for various reasons:
 
 As soon as you introduce breaking changes, they might decide to stick with the
 last working version indefinitely. As an example of a popular library that took
-a long time to upgrade, Hadoop introduced Guava 11.0.2 in May of 2012, and
+a long time to upgrade, Hadoop had introduced Guava 11.0.2 in May of 2012, and
```

Comment:
don't need "had" here

Reply:
Sorry, this file is not to be in the PR.